<launch>
  <arg name="rho_limit" default="95" />
  <arg name="baseline"  default="0.013" />

  <node pkg="ricoh_camera" type="project_eqrect.py" name="project" output="screen">
    <param name="camchain" value="$(find ricoh_camera)/config/camchain-stereo_2019-07-05-08-24-01.yaml"/>
    <param name="front_cam_id"  value="cam3"/>
    <param name="back_cam_id"   value="cam2"/>
    <param name="rho_limit" value="$(arg rho_limit)" />
    <param name="baseline"  value="$(arg baseline)" />
  </node>
  <node name="image_eqrect_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="image_rect" />
  </node>

</launch>
